src/ace.F90

```diff
@@ -1019,10 +1012,9 @@ end subroutine read_energy_dist
 !===============================================================================
 
   recursive subroutine get_energy_dist(edist, loc_law, delayed_n)
-
-    type(DistEnergy), pointer :: edist     ! energy distribution
-    integer, intent(in)       :: loc_law   ! locator for data
-    logical, optional         :: delayed_n ! is this for delayed neutrons?
+    type(DistEnergy), intent(inout) :: edist     ! energy distribution
```

Comment:
This might be @nelsonag referred to with regards to the comment alignment. I tend to like aligned comments too, but don't see it as a necessity.